packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -861,6 +864,8 @@ class _ReorderableItemState extends State<_ReorderableItem> {
       setState(() {
         _dragging = dragging;
       });
+    } else {
```

Comment:
We generally don't produce console warnings like this.  Assuming that asserting in this case is not warranted, I think it would be best to leave the message out.

Reply:
D'oh! This was a stray debug message that I used to diagnose the problem. Missed taking it out when I was submitting. Thx!